src/www/_layouts/layout-docs.njk

```diff
@@ -0,0 +1,65 @@
+{% extends "www/_layouts/_master-layout.njk" %}
+{% set section = section if section else "Components" %}
+
+{% block body %}
+<div class="wmnds-grid wmnds-grid--spacing-lg-2-lg">
+  <div class="wmnds-col-1 wmnds-col-lg-1-3 wmnds-p-b-lg">
+
+    <h4>Installation</h4>
+    <ul class="wmnds-website-subnav">
+      <li>
+        <a href="/docs/get-started/" class="wmnds-link">Get started</a>
+      </li>
+    </ul>
+
+    <hr class="wmnds-col-1 wmnds-col-sm-3-4"/>
+
+    <h4>Setup Guides</h4>
+    <ul class="wmnds-website-subnav">
+      <li>
+        <a href="/docs/get-started/prototype-kit/" class="wmnds-link">Prototype Kit</a>
+      </li>
+      <li>
+        <a href="/docs/get-started/production/" class="wmnds-link">Production</a>
+      </li>
+      <li>
+        <a href="/docs/get-started/mobile-application/" class="wmnds-link">Mobile application</a>
+      </li>
+      <li>
+        <a href="/docs/get-started/design/" class="wmnds-link">Design</a>
+      </li>
+  </ul>
+
+  <hr class="wmnds-col-1 wmnds-col-sm-3-4"/>
+
+  <h4>Usage</h4>
+  <ul class="wmnds-website-subnav">
+    <li>
+      <a href="/docs/get-started/using-the-design-system/" class="wmnds-link">Using the design system</a>
+    </li>
+  </ul>
+
+  <hr class="wmnds-col-1 wmnds-col-sm-3-4"/>
+
+  <h4>Other</h4>
+  <ul class="wmnds-website-subnav">
+    <li>
+      <a href="/user-research/" class="wmnds-link">User research</a>
```

Comment:
Having this page opening in a different template bugged me a little. 
If we can't change the template maybe we could change the link to open in a new tab...